src/functionalTest/resources/scenarios/various-methods-for-http-action/tasks.conf

```diff
@@ -0,0 +1,136 @@
+global.handler.fragmentsHandler.config {
+  tasks {
+    various-methods-for-http-action {
+      action = get
```

Comment:
You probably can run all http actions in parallel.
On the other hand it really doesn't matter in the test.

Reply:
Good catch - running in parallel will make the task a bit faster. I'll fix it.